Client/Api/Builder/ICamundaCloudClientBuilder.cs

```diff
@@ -76,6 +76,13 @@ public interface ICamundaCloudClientBuilderFinalStep
         /// <returns>the next step in building a ICamundaCloudClient</returns>
         ICamundaCloudClientBuilderFinalStep UseAuthServer(string url);
 
+        /// <summary>
+        /// Sets the given path to store credentials on disk.
+        /// </summary>
+        /// <param name="path">the path where to store the credentials.</param>
+        /// <returns>the fluent ICamundaCloudClientBuilderFinalStep.</returns>
+        ICamundaCloudClientBuilderFinalStep UsePersistedStoragePath(string path);
```

Comment:
Thought it we be more ideal to be more explicit with what path you're settings.

Reply:
Not sure what you mean here?

Reply:
Just meant thought it would be better than passing forward the `UsePath` name.